codegens/csharp-restsharp/lib/parseRequest.js

```diff
@@ -41,6 +41,24 @@ function parseContentType (request) {
   return request.getHeaders({enabled: true, ignoreCase: true})['content-type'] || 'text/plain';
 }
 
+/**
+ *
+ * @param {Object} requestBody - JSON object representing body of request
+ * @returns {String} code snippet for graphql body
+ */
+function parseGraphQL (requestBody) {
+  let query = requestBody.graphql.query,
+    graphqlVariables;
+  try {
+    graphqlVariables = JSON.parse(requestBody.graphql.variables);
+  }
+  catch (e) {
+    graphqlVariables = {};
+  }
```

Comment:
Why is this required?

Reply:
To catch the error from JSON parsing. Just sending an empty object for variables. Not to send any undefined value or something.